launchpad/launch/run_locally/launch_local_output_to_files.py

```diff
@@ -20,8 +20,11 @@
 
 from launchpad.launch import worker_manager
 
-_LOGGING_DIR = '/tmp/launchpad_out/'
-
+# Use environment variable to direct logging to specified directory.
+if 'LAUNCHPAD_LOGGING_DIR' in os.environ:
+    _LOGGING_DIR = os.environ['LAUNCHPAD_LOGGING_DIR']
+else:
+    _LOGGING_DIR = '/tmp/launchpad_out/'
```

Comment:
Could you add a `logging.info()` (or `print()`) on top to explain that it can be customized? E.g., "Logs are being output to /tmp/launchpad_out/. This can be customized by setting the LAUNCHPAD_LOGGING_DIR environment variable."

Reply:
That's a good idea @ddmbr. I added a print statement inside the function. In this way the message is printed right before the other "Logging to: ..." messages in that function.